recipes/python-periodictable/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-periodictable" %}
```

Comment:
I think historically there had been discussion about making package names for python begin with `python-`, but it was abandoned. Under almost all circumstances the name matches the pypi package name.